curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/655b49333d9f265bc1512152.md

```diff
@@ -7,11 +7,11 @@ dashedName: step-87
 
 # --description--
 
-Notice that the album art in the HTML and songs in the `allSongs` array have changed. We've swapped out the original songs for shorter ones that you can use to test your app in the upcoming steps.
+Notice that the album art in the HTML and songs in the `userData.songs` array have changed. We've swapped out the original songs for shorter ones that you can use to test your app in the upcoming steps.
 
 Next, you need to check if there is a next song to play. Retrieve the current song index by calling the `getCurrentSongIndex()` function, and save it in a `currentSongIndex` constant.
```

Comment:
@jdwilkin4 will it be a good idea ?
```suggestion
Next, you need to check if there is a next song to play by retrieving the current song index by calling the `getCurrentSongIndex()` function, and save it in a `currentSongIndex` constant.
```

Reply:
I personally like the original version better with the two separate sentences

Reply:
@jdwilkin4  I'll follow your preference then, I'm dropping this and approving the PR